bedrock/base/templates/includes/protocol/navigation/index.html

```diff
@@ -24,12 +24,14 @@
         {% if not hide_nav_download_button %}
         <div class="mzp-c-navigation-download">
           {{ download_firefox(alt_copy=_('Download Firefox'), dom_id='protocol-nav-download-firefox', button_color='mzp-t-secondary mzp-t-small', download_location='nav') }}
```

Comment:
Suggest wrapping this in `{% if not hide_nav_download_button %}`, so that it only hides the download button?